lms/static/js/discovery/views/courses_listing.js

```diff
@@ -31,12 +31,15 @@
             },
 
             renderItems: function() {
+                /* eslint no-param-reassign: [2, { "props": true }] */
```

Comment:
Just b/c I'm curious: Why is this done as a start-stop instead of a single line comment for eslint?